custom_components/sems/sensor.py

```diff
@@ -8,26 +8,25 @@
 import json
 import logging
 
-from bs4 import BeautifulSoup
 import requests
 import voluptuous as vol
 
 from homeassistant.components.sensor import PLATFORM_SCHEMA
-from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, CONF_ID
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.entity import Entity
 
-#REQUIREMENTS = ['BeautifulSoup4==4.7.0']
-
 # Validation of the user's configuration
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_USERNAME): cv.string,
-    vol.Required(CONF_PASSWORD): cv.string
+    vol.Required(CONF_PASSWORD): cv.string,
+    vol.Required(CONF_ID): cv.string
```

Comment:
I would prefer use of station_id instead of ID. It is more self describing.

CONF_STATION_ID